plugins/inputs/fail2ban/README.md

```diff
@@ -60,7 +60,7 @@ Defaults!FAIL2BAN !logfile, !syslog, !pam_session
 
 ## Example Output
 
-```shell
+```text
 # fail2ban-client status sshd
 Status for the jail: sshd
```

Comment:
This is actually a command and its output, not sure what to use here..

Reply:
This block describes the result of directly executing the binary, I have temporarily explained and reserved. https://github.com/influxdata/telegraf/pull/12989/commits/f1af219b87a67d75395bb4c9be539631df9450e1

Reply:
Yeah, I'm just not sure what to do here.. Split into a `shell` and a `text` block? Keep both in the `text` block? Or keep this a `shell` block?

@powersj thoughts?